src/ansys/simai/core/data/training_data.py

```diff
@@ -269,3 +284,12 @@ def upload_folder(
         if compute:
             self._client._api.compute_training_data(training_data_id)
         return uploaded_parts
+
+    @property
+    def subset_names(self) -> List[str]:
```

Comment:
That's not needed since we've got the `SubsetEnum` type.
I think you wrote this because of a previous comment that was misguided by the fact that when reading the previous code, reviewers (me included) had the impressions that subsets where being created and not just assigned (hense the need to list them) -> it's not the case so we don't need this function.

Reply:
If the intention is to use it like:

```py
from ansys.simai.core.data.types import SubsetEnum

td.assign_subset("project_id", SubsetEnum.VALIDATION)
```

then, indeed, we don't need to export it through a `TrainingData` property. My comment was more on securing the intended usage, than requesting a code modification 😅